CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 Breaking changes:
 
+- [core] renamed method `registerComositionEventListeners()` has been renamed to `registerCompositionEventListeners()` [#6912](https://github.com/eclipse-theia/theia/pull/6912)
```

Comment:
[minor] the wording is a little off, `renamed` is mentioned twice. Something like the following would be more appropriate:

```md
- [core] renamed method `registerComositionEventListeners()` to `registerCompositionListeners()`...
```